float64 center_x
float64 center_y
float64 center_radius

float64[] particle_x
float64[] particle_y
float64   particle_radius
